packages/cypress/src/support.ts

```diff
@@ -51,6 +51,10 @@ afterEach(() => {
               }),
             },
             pageUrl: url,
+            viewport: {
+              height: Cypress.config('viewportHeight'),
+              width: Cypress.config('viewportWidth'),
```

Comment:
This seems to be the only way to get the viewport size, however, this seems to only reflect the real values if the viewport is set in one of these two ways:
- https://docs.cypress.io/api/commands/viewport#Reset-viewport-via-Cypressconfig
- https://docs.cypress.io/api/commands/viewport#Set-viewport-in-the-test-configuration

Notably, using `cy.viewport()` to set it [as described here](https://docs.cypress.io/api/commands/viewport#Organize-desktop-vs-mobile-tests-separately) does not work since the `Cypress.config` values will stay set to whatever they default to.